vllm/config.py

```diff
@@ -300,6 +303,17 @@ def _verify_cuda_graph(self) -> None:
         self.max_seq_len_to_capture = min(self.max_seq_len_to_capture,
                                           self.max_model_len)
 
+    def _verify_output_proc_callback(self) -> None:
+        logger.warning(
+                    "To see benefits of output processor callback, enable CUDA "
+                    "graph. Since, enforce-eager is enabled, output processor callback "
+                    "cannot be used")
+        self.use_output_proc_callback = not self.enforce_eager
+        # TO DO: assert no beam search
+        # TO DO: assert no pipeline parallelism
+        # TO DO: assert no spec decoding
```

Comment:
Supporting spec decoding will need more thought. Mostly because, `execute_model` inside spec dec's step is quite different from non-spec dec codepath and we need to figure out the right place to call the callback. Pipeline parallelism should be easier, and can be done in follow-up PR.

Reply:
> Supporting spec decoding will need more thought. Mostly because, `execute_model` inside spec dec's step is quite different from non-spec dec codepath and we need to figure out the right place to call the callback. Pipeline parallelism should be easier, and can be done in follow-up PR.

Hi, we have better performance with  Asynchronous Output Processor,  do we have any schedule to support async in spec dec's path? thanks